frontend/src/Styles/Themes/dark.js

```diff
@@ -209,7 +209,7 @@ module.exports = {
 
   calendarTodayBackgroundColor: '#3e3e3e',
   calendarBackgroundColor: '#2a2a2a',
-  calendarBorderColor: '#cecece',
+  calendarBorderColor: ''#393f45',
```

Comment:
Need to remove the extra quote here

Reply:
Oh wow, nice rebase fail here, thank you!